docs/configuration/single-process-config.yaml

```diff
@@ -33,6 +33,11 @@ ingester:
   spread_flushes: true
   chunk_age_jitter: 0
 
+  walconfig:
+    wal_enabled: true
+    recover_from_wal: true
+    wal_dir: /tmp/cortex/wal
```

Comment:
Running Cortex with the default image fails with the following error:
`failed to recover from WAL: open /tmp/cortex/wal: no such file or directory`

This is not just an error you get when running integration tests but also running it locally with (the following command picked from the doc):
`./cortex -config.file=./docs/configuration/single-process-config.yaml`

Reply:
Yes, I found that out before seeing this review and have already pushed a fix for it. Now a directory is created when WAL or recovery from WAL is enabled.

Reply:
Integration tests are passing now :)